apps/acceptance/homescreen/regions/homescreen.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+var View = require('../../lib/view');
+var Marionette = require('marionette-client');
+
+
+function Homescreen(client) {
+    View.call(this, client);
+}
+
+Homescreen.prototype = Object.create(View.prototype);
+Homescreen.prototype.constructor = Homescreen;
```

Comment:
Why do we need to specify the constructor, here? I though it was implied when you define the function `Homescreen`